wiz-admission-controller/values.yaml

```diff
@@ -151,6 +151,52 @@ imageIntegrityWebhook:
 
   patchImageDigestAnnotation: true # If true, annotate the pod with mapping between tag->digest.
 
+# Kubernetes audit logs webhook is currently in private preview.
+# This webhooks sends the audit logs to Wiz, and should never block any requests.
+kubernetesAuditLogsWebhook:
+  enabled: false 
+
+    # Override to run admission controller on specific resources.
+  rules: # https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#matching-requests-rules
+    - operations:
+      - "*"
+      apiGroups:
+      - "*"
+      apiVersions:
+      - "*"
+      resources:
+        - "namespaces"
+        - "secrets"
+        - "configmaps"
+        - "storageclasses"
+        - "persistentvolumes"
+        - "persistentvolumeclaims"
+        - "controllerrevisions"
+        - "ingresses"
+        - "services"
+        - "networkpolicies"
+        - "pods"
+        - "deployments"
+        - "jobs"
+        - "cronjobs"
+        - "replicasets"
+        - "statefulsets"
+        - "daemonsets"
+  namespaceSelector:  # https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#matching-requests-namespaceselector
+    matchExpressions:
+    - key: kubernetes.io/metadata.name # Using "kubernetes.io/metadata.name" key to exclude specific namespaces (https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/2161-apiserver-default-labels/README.md#proposal)
+      operator: NotIn
+      values:
+      - kube-system
```

Comment:
Ar you sure we don't want to monitor reviews from kube system?

Reply:
Yes, https://wiz-sec.slack.com/archives/C05HCDVJMTN/p1710083199005799?thread_ts=1710078245.159749&cid=C05HCDVJMTN
And I verified this with him.